libmscore/measure.cpp

```diff
@@ -2513,31 +2513,11 @@ bool Measure::isFirstInSystem() const
 
 void Measure::scanElements(void* data, void (* func)(void*, Element*), bool all)
 {
-    MeasureBase::scanElements(data, func, all);
-
-    int nstaves = score()->nstaves();
-    for (int staffIdx = 0; staffIdx < nstaves; ++staffIdx) {
-        if (!all && !(visible(staffIdx) && score()->staff(staffIdx)->show())) {
-            continue;
-        }
-        MStaff* ms = _mstaves[staffIdx];
-        func(data, ms->lines());
-        if (ms->vspacerUp()) {
-            func(data, ms->vspacerUp());
-        }
-        if (ms->vspacerDown()) {
-            func(data, ms->vspacerDown());
-        }
-        if (ms->noText()) {
-            func(data, ms->noText());
-        }
-    }
-
-    for (Segment* s = first(); s; s = s->next()) {
-        if (!s->enabled()) {
+    for (ScoreElement* el : (*this)) {
+        if (el == nullptr || el->isMeasure()) { // do not scan Measures 'inside' mmrest measure
             continue;
```

Comment:
Move comment to next line to avoid using so many columns. You can put it on the same line as the `continue` since it will still be under 80 chars. (We allow [up to 120 chars](https://github.com/musescore/MuseScore/blob/41aedce5e42ccb67102988798ffab5cc9eff1658/tools/codestyle/uncrustify_musescore.cfg#L1285) but there's no reason to exceed 80 where it is easily avoidable.)